olm/observability-operator-index/index.yaml

```diff
@@ -0,0 +1,91 @@
+  mediatype: image/svg+xml
+name: observability-operator
+schema: olm.package
+---
+schema: olm.channel
+package: observability-operator
+name: development
+entries:
+  - name: observability-operator.v0.0.13
+### DEVELOPMENT_CHANNEL_MARKER ###
```

Comment:
Once this patch merges and we can release 0.0.14 successfully, we should probably revisit this part and see if we can cause kustomize to add the entries. 
For every new version we add it should add something like 

```
  - name: observability-operator.v<new-version>
    replaces: observability-operator.v<previous-version>
```

to appropriate channels.

Reply:
:+1:  yep was thinking the same when I looked at `update-channels.sh` :) 
If kustomize doesn't work maybe at least something else that actually interprets yaml.

Reply:
I have seen other teams write (python) script to parse and update yaml 
but I think maintaining separate files for channels and using kustomize to assemble it might work. I took the easy route (as you can see :D )